editor/src/components/canvas/canvas-utils.ts

```diff
@@ -1934,13 +1935,22 @@ export function getParseSuccessForFilePath(
   }
 }
 
+export type RemixValidPathsGenerationContext =
+  | { type: 'disabled' }
+  | {
+      type: 'active'
```

Comment:
Surely this should be either `'disabled'` and `'enabled'`, or `'inactive'` and `'active'` 😁